.vs/config/applicationhost.config

```diff
@@ -171,7 +171,7 @@
             </site>
             <site name="Commencement.Mvc" id="3">
                 <application path="/" applicationPool="Clr4IntegratedAppPool">
-                    <virtualDirectory path="/" physicalPath="C:\Users\postit\Documents\GitHub\Commencement\Commencement.Mvc" />
+                    <virtualDirectory path="/" physicalPath="C:\GitProjects\Commencement\Commencement.Mvc" />

```

Comment:
@jSylvestre not too important but you should add `.vs/` to the .gitignore since we don't need to be tracking these files